pkg/component/etcd/etcd.go

```diff
@@ -584,6 +603,67 @@ func (e *etcd) emptyHVPA() *hvpav1alpha1.Hvpa {
 	return &hvpav1alpha1.Hvpa{ObjectMeta: metav1.ObjectMeta{Name: e.etcd.Name, Namespace: e.namespace}}
 }
 
+func (e *etcd) emptyVerticalPodAutoscaler() *vpaautoscalingv1.VerticalPodAutoscaler {
+	return &vpaautoscalingv1.VerticalPodAutoscaler{ObjectMeta: metav1.ObjectMeta{Name: e.etcd.Name + "-vpa", Namespace: e.namespace}}
```

Comment:
```suggestion
	return &vpaautoscalingv1.VerticalPodAutoscaler{ObjectMeta: metav1.ObjectMeta{Name: e.etcd.Name, Namespace: e.namespace}}
```

We don't need the `-vpa` suffix. VerticalPodAutoscaler is already the kind. For example we don't suffix with `-deploy` Deployments, with `-svc` Services, etc. Most probably the `-vpa` suffix exists in our systems because someone initially added it and then everyone did copy/paste.

Reply:
done.